src/corelibs/U2Formats/src/tasks/ExportTasks.h

```diff
@@ -42,6 +42,13 @@ class U2FORMATS_EXPORT ExportAlignmentTask : public DocumentProviderTask {
 
     void run() override;
 
+    /**
+     * Maximum MSA size in bytes that can be loaded/stored by UGENE safely.
+     * UGENE loads a full MSA model into a memory in a lot of places across the codebase.
+     * This constant should be limit the maxim size of MSA produced by UGENE, so UGENE won't create objects it can't handle.
+     */
+    static constexpr qint64 MAX_SAFE_ALIGNMENT_SIZE_TO_EXPORT = 100 * 1000 * 1000;
```

Comment:
"100 * 1024 * 1024" is a bit more correct

Reply:
When I see powers of 2 I start thinking about low-level platform limitation: how many values in int32 or like that.

In bioinformatics we do not use powers of two: 1kbp is equal to 1000bp - all our code is written that way (see rulers, coordinate formatters etc.)

Here this limitation is not a platform limitation, but just a fact that for some big values UGENE becomes very slow.
100M here is not bytes. We can only approximately estimate how much memory is needed to store an alignment with 100M characters: it can be 100Mb or 200Mb or even 10Gb